src/hooks/useUploadsManager/useUploadsManager.tsx

```diff
@@ -13,16 +12,15 @@ export const UploadManagerProvider: React.FC = ({ children }) => {
   const [uploadsProgress, setUploadsProgress] = useState<UploadsProgressRecord>({})
 
   const startFileUpload = useCallback(
-    async (file: File | Blob, asset: InputAssetUpload) => {
+    async (file: File | Blob, asset: InputAssetUpload, storageNodeUrl: string) => {
       const setAssetUploadProgress = (progress: number) => {
         setUploadsProgress((prevState) => ({ ...prevState, [asset.contentId]: progress }))
       }
 
       try {
         addAsset({ ...asset, lastStatus: 'inProgress', size: file.size })
         setAssetUploadProgress(0)
-
-        const assetUrl = new URL(asset.contentId, STORAGE_NODE_URL)
+        const assetUrl = new URL(asset.contentId, storageNodeUrl + '/asset/v0/')
```

Comment:
Let's put that suffix somewhere in the `@/config`

Reply:
I think maybe we should extract this full line somewhere because we use it in two places. Let's just make it a separate function in `@/utils/assets`